_automation/grammars.json

```diff
@@ -0,0 +1,275 @@
+[
+  {
+    "language": "bash",
+    "url": "https://github.com/tree-sitter/tree-sitter-bash",
+    "files": [
+      "parser.c",
+      "scanner.cc"
+    ],
+    "reference": "v0.19.0",
+    "revision": "b6667bed364733c8a8f8e5973749f86cfa04ba2a"
+  },
+  {
+    "language": "c",
+    "url": "https://github.com/tree-sitter/tree-sitter-c",
+    "files": [
+      "parser.c"
+    ],
+    "reference": "v0.20.2",
+    "revision": "7175a6dd5fc1cee660dce6fe23f6043d75af424a"
+  },
+  {
+    "language": "csharp",
+    "url": "https://github.com/tree-sitter/tree-sitter-c-sharp",
+    "files": [
+      "parser.c",
+      "scanner.c"
+    ],
+    "reference": "v0.20.0",
+    "revision": "7a47daeaf0d410dd1a91c97b274bb7276dd96605"
+  },
+  {
+    "language": "cpp",
+    "url": "https://github.com/tree-sitter/tree-sitter-cpp",
+    "files": [
+      "parser.c",
+      "scanner.cc"
+    ],
+    "reference": "v0.20.0",
+    "revision": "f44509141e7e483323d2ec178f2d2e6c0fc041c1"
+  },
+  {
+    "language": "css",
+    "url": "https://github.com/tree-sitter/tree-sitter-css",
+    "files": [
+      "parser.c",
+      "scanner.c"
+    ],
+    "reference": "v0.19.0",
+    "revision": "9668e889161358d18a558f68f1614b6edde34029"
+  },
+  {
+    "language": "cue",
+    "url": "https://github.com/eonpatapon/tree-sitter-cue",
+    "files": [
+      "parser.c",
+      "scanner.c"
+    ],
+    "reference": "main",
+    "revision": "4ffcda8c2bdfee1c2ba786cd503d0508ea92cca2"
```

Comment:
When doing a forced "update-all" where all grammars are reinstalled, then there are changes concerning the cue bindings. So, I assume the revision is not correct.

Reply:
Thanks! Yes, I committed incorrect revisions, sorry.

P.S. It would still show you diff but if you look at it with `-w` flag to ignore space changes you will notice only formatting & comments are different now. It will be fixed in separate PRs when updating grammars.